app/ui/tslint.json

```diff
@@ -207,19 +213,39 @@
     ],
     "component-selector": [
       "element",
-      ["syndesis"],
+      [
+        "syndesis"
+      ],
       "kebab-case"
     ],
     "directive-class-suffix": true,
     "directive-selector": [
       "attribute",
-      ["syndesis", "syn"],
+      [
+        "syndesis",
+        "syn"
+      ],
       "camelCase"
     ],
     "import-destructuring-spacing": true,
-    "pipe-naming": ["camelCase"],
+    "pipe-naming": [
+      "camelCase"
+    ],
     "use-host-property-decorator": true,
     "use-input-property-decorator": true,
-    "use-output-property-decorator": true
+    "use-output-property-decorator": true,
+    // ***************************************************
+    // TSLint rules for RxJS
+    // ***************************************************
+    "rxjs-add": {
+      "options": [
+        {
+          "allowElsewhere": false,
+          "allowUnused": false,
+          "file": "rxjs.imports.ts"
+        }
+      ],
+      "severity": "error"
+    }
   }
-}
+}
```

Comment:
Why the overall reformatting of the JSON file? splitting one or two-items arrays into 4 LOC makes the whole thing more hardly readable indeed

Reply:
oh, I think my editor did that, I can reset those parts...